RELEASE-NOTES.md

```diff
@@ -1,3 +1,59 @@
+# PatternFly Albatross Milestone (2019-07-02)
+- @patternfly/patternfly@2.17.0
+
+**Components:**
+*   **About modal box**: Removed the title classes from the strapline paragraph. Changed strapline `font-size` to 14px. ([#1951](https://github.com/patternfly/patternfly-next/pull/1951))
```

Comment:
nit - there are a handful of places where the colon isn't bold. `**:` instead of `:**`